src/utils.js

```diff
@@ -125,3 +125,8 @@ export function cssgu(strings, ...substitutions) {
       return isNaN(value) ? match : `${value * GU}px`
     })
 }
+
+export const iOS =
```

Comment:
We have this in `src/components/Preferences/LocalIdentities.js`. We don't have to change this here, but let's not forget to change this in the `new-styles` branch :).

Reply:
Yep, I'm aware, I added it there and now it has been moved [here](https://github.com/aragon/aragon/blob/feature/global-settings/src/utils.js#L129)